here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/DefaultStorageHandler.java

```diff
@@ -80,31 +60,30 @@ public DefaultStorageHandler(
     super(hub);
     this.eventHandler = eventHandler;
     this.eventTarget = eventTarget;
-    this.properties = JsonSerializable.convert(eventHandler.getProperties(), DefaultStorageHandlerProperties.class);
+    this.properties = Objects.requireNonNull(
+            JvmProxyUtil.box(eventHandler.getProperties(), DefaultStorageHandlerProperties.class));
```

Comment:
Our previous casting using `Json` util is annotated with not null, but actually Jackson underneath does not also guarantee that. With this check it will throw an NPE upfront.